vercel.json

```diff
@@ -1424,5 +1424,20 @@
             "source": "/docs/surveys/webhook",
             "destination": "/docs/surveys/destinations"
         }
+    ],
+    "headers": [
+        {
+            "source": "/(.*)",
+            "headers": [
+                {
+                    "key": "Content-Security-Policy-Report-Only",
+                    "value": "default-src 'self'; script-src 'self' 'unsafe-inline' https://*.posthog.com; style-src 'self' 'unsafe-inline'; img-src 'self' data: https://*.posthog.com https://res.cloudinary.com https://cdn.jsdelivr.net; font-src 'self' data: https://d27nj4tzr3d5tm.cloudfront.net; connect-src 'self' https://*.posthog.com https://api.github.com https://lottie.host; frame-src 'self'; report-uri https://d1iyf54ud053a4.cloudfront.net/csp?token=sTMFPsFhdP1Ssg&sample_rate=0.1&v=1; report-to csp-endpoint"
```

Comment:
```suggestion
                    "value": "default-src 'self'; script-src 'self' 'unsafe-inline' https://*.posthog.com; style-src 'self' 'unsafe-inline'; img-src 'self' data: https://*.posthog.com https://res.cloudinary.com font-src 'self' data: https://d27nj4tzr3d5tm.cloudfront.net; connect-src 'self' https://*.posthog.com https://api.github.com https://lottie.host; frame-src 'self'; report-uri https://d1iyf54ud053a4.cloudfront.net/csp?token=sTMFPsFhdP1Ssg&sample_rate=0.1&v=1; report-to csp-endpoint"
```

Reply:
let's remove `https://cdn.jsdelivr.net; ` so that we're definitely reporting violations, since we need to see this fail and then recover in the tracking